lib/controllers/widget/hardware_os_card.dart

```diff
@@ -0,0 +1,405 @@
+                              )
+                            ],
+                          )),
+                      Padding(
+                          padding: EdgeInsets.only(top: 10, bottom: 10),
+                          child: Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: <Widget>[
+                              Text(
+                                'Mac Address',
+                                style: TextStyle(
+                                    fontSize: 18,
+                                    color: Colors.black87,
+                                    fontWeight: FontWeight.w500),
+                                overflow: TextOverflow.clip,
+                              ),
+                              Text(
+                                widget._device.macAddress ?? '-',
+                                style: TextStyle(
+                                    fontSize: 16, color: Colors.black54),
+                                overflow: TextOverflow.clip,
+                              )
+                            ],
+                          )),
+                      Padding(
+                          padding: EdgeInsets.only(top: 10, bottom: 10),
+                          child: Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: <Widget>[
+                              Text(
+                                'Last Known Network',
+                                style: TextStyle(
+                                    fontSize: 18,
+                                    color: Colors.black87,
+                                    fontWeight: FontWeight.w500),
+                                overflow: TextOverflow.clip,
+                              ),
+                              Text(
+                                widget._device.lastKnownNetwork == null
+                                    ? '-'
+                                    : widget._device.lastKnownNetwork
+                                        .map((o) => o.ipAddress)
+                                        .join('\n'),
+                                style: TextStyle(
+                                    fontSize: 16, color: Colors.black54),
+                                overflow: TextOverflow.clip,
+                              )
+                            ],
+                          )),
+                      Padding(
+                          padding: EdgeInsets.only(top: 10, bottom: 10),
+                          child: Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: <Widget>[
+                              Text(
+                                'OS version',
+                                style: TextStyle(
+                                    fontSize: 18,
+                                    color: Colors.black87,
```

Comment:
Since most of these TextStyles are repeated can we have a variable that has it and we use that instead?

Reply:
I refactored the UI in another pull request.